lib/util/index.js

```diff
@@ -541,6 +541,21 @@ function createMediaSignalingPayload(dominantSpeaker, networkQuality) {
       : {});
 }
 
+/**
+ * Create the Subscribe payload included in an RSP connect message.
```

Comment:
@makarandp0 In the future, this can also be part of an RSP "update" message. So, can you say "... in an RSP connect/update message?